mobile/lib/src/infrastructure/sleep_history/sleep_history_repository.dart

```diff
@@ -21,13 +21,22 @@ class SleepHistoryRepository implements ISleepHistoryRepository {
   }
 
   @override
-  void selectNight(NightStats stat) {
+  void viewNight(NightStats stat) {
```

Comment:
Pourquoi viewNight et watch? Pourquoi pas simplement avoir les opérations CRUD dans notre repository?

Reply:
J'ai changé le nom pour selectSleepSequence, mais je sais pas si c'est vraiment mieux. Ce qui se passe c'est que le history cubit fait une selection sur la sequence qu'on veut voir, ce que le répertoire ajoute à un sequenceStreamController. La page de visualisation écoute ce stream et update automatiquement sont contenu. Je trouvais que c'était le moyens le plus simple d'update la page de stats.

Reply:
Il me semble que pour sélectionner les nuits on a pas besoin de passer par le DAL. On aura seulement besoin d'utiliser la méthode delete de notre repo pour supprimer les sequences

Reply:
Should be fine now, let me know!

Reply:
C'est parfait!